Analyzer/plugins/Analyzer.cc

```diff
@@ -12,168 +12,12 @@
 // Modifications by Dylan Angie Frank Apparu
 //                  and Tamas Almos Vami
 
-// v19p0
-// - change double to float
-// - create fillDescription
```

Comment:
All this can go to the `README` from now on let's just keep the most recent changes in here